src/item.h

```diff
@@ -670,6 +677,33 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
      */
     int chip_resistance( bool worst = false ) const;
 
+    /** How much damage has the item sustained? */
+    int damage() const {
+        return damage_;
+    }
+
+    /** Minimum amount of damage to an item (state of maximum repair) */
+    int min_damage() const {
+        return MIN_ITEM_DAMAGE;
+    }
+
+    /** Maximum amount of damage to an item (state before destroyed) */
+    int max_damage() const {
+        // Following locations remain dependent on MAX_ITEM_DAMAGE == 4
+        // * player::sort_armor()
+        // * material_type::_dmg_adj
+        // * item::tname()
+        return MAX_ITEM_DAMAGE;
+    }
+
+    /**
+     * Apply damage to item constrained by @ref min_damage and @ref max_damage
+     * @param qty maximum amount by which to adjust damage (negative permissible)
+     * @param dmg type of damage which may be passed to @ref on_damage callback
+     * @return whether item should be destroyed
+     */
+    bool mod_damage( int qty = 1, damage_type dmg = DT_NULL );
```

Comment:
`mod_damage` shouldn't have a default `qty`. A more explicit `inc_damage( damage_type )` would be better.